src/studio/AppTemplates/AspNet/App/Startup.cs

```diff
@@ -162,6 +163,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 });
             }
 
+            app.UseDefaultSecurityHeaders();
```

Comment:
Shouldn't this change be done in both templates?

Reply:
Its already in the other template.

Reply:
oh, sorry